src/index.ts

```diff
@@ -24,48 +23,49 @@ const {
   divide,
   sub,
   eq,
-  timing,
-  Node,
+  or,
 } = Animated;
 
-export { timing, clockRunning, add };
-
-export type TimingConfig = Parameters<typeof timing>[1];
-export type Clock = Parameters<typeof clockRunning>[0];
-export type Node = ReturnType<typeof add>;
-export type Adaptable<T> = Node | T;
-
 // ## Animations
-export const snapPoint = (value: Adaptable<number>, velocity: Adaptable<number>, points: number[]) => {
+export const snapPoint = (
+  value: Animated.Adaptable<number>,
+  velocity: Animated.Adaptable<number>,
+  points: number[],
+) => {
   const point = add(value, multiply(0.2, velocity));
-  const diffPoint = (p: Adaptable<number>) => abs(sub(point, p));
+  const diffPoint = (p: Animated.Adaptable<number>) => abs(sub(point, p));
   const deltas = points.map(p => diffPoint(p));
   const minDelta = min(...deltas);
-  return points.reduce((acc: Node, p: number) => cond(eq(diffPoint(p), minDelta), p, acc), new Value());
+  return points.reduce((acc: Animated.Node<any>, p: number) => cond(eq(diffPoint(p), minDelta), p, acc), new Value());
 };
 
 export const binaryInterpolation = (
-  value: Adaptable<number>,
-  origin: Adaptable<number>,
-  destination: Adaptable<number>,
+  value: Animated.Adaptable<number>,
+  origin: Animated.Adaptable<number>,
+  destination: Animated.Adaptable<number>,
 ) => interpolate(value, {
   inputRange: [0, 1],
   outputRange: [origin, destination],
 });
 
-export const lookup = (
-  array: Adaptable<number>[],
-  index: Adaptable<number>,
-  notFound: Node = new Value(),
+export const find = (
```

Comment:
You should probably change README.md also

Reply:
yes and yes, thank you :)

Reply:
I will add the breaking change note in the squash commit.